ledger/participant-integration-api/src/docs/scala/com/daml/platform/docs/SelfServiceErrorCodes_MigrationGuideGen_App.scala

```diff
@@ -3,149 +3,43 @@
 
 package com.daml.platform
 
+import java.nio.charset.StandardCharsets
+
+import com.daml.error.{Description, ErrorCategory, Resolution, RetryStrategy}
 import io.circe.{Json, JsonObject, ParsingFailure, yaml}
 import org.apache.commons.io.IOUtils
 
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
-import java.nio.charset.StandardCharsets
-
-import com.daml.error.{Description, ErrorCategory, Resolution, RetryStrategy}
 
 // TODO error codes: Delete it once final version of migration guide is ready.
 /** How to update self-service error codes migration guide table:
-  * 1. Make appropriate changes to self-service-error-codes-migration.yml.
-  * 2. Run this app to generate reST table in stdout.
-  * 3. Paste that table into docs/source/error-codes/self-service/index.rst.
-  */
+ * 1. Make appropriate changes to self-service-error-codes-migration.yml.
+ * 2. Run this app to generate reST table in stdout.
+ * 3. Paste that table into docs/source/error-codes/self-service/index.rst.
+ */
 object SelfServiceErrorCodes_MigrationGuideGen_App {
```

Comment:
I think this is a good time to remove this class

Reply:
Planning to do that in https://digitalasset.atlassian.net/browse/DPP-762